modules/account-lib/src/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -0,0 +1,90 @@
+import * as paillierBigint from 'paillier-bigint';
+import { Secp256k1Curve } from '../../curves';
+import Shamir from '../../shamir';
+import { bigIntToHex } from '../../util';
+import { NShare, PShare, KeyShare, KeyCombined } from './types';
+
+export default class Ecdsa {
+  static curve: Secp256k1Curve = new Secp256k1Curve();
+  static shamir: Shamir = new Shamir(Ecdsa.curve);
+  /**
+   * Generate shares for player at index and split keys `(t,n)` ways.
+   * @param {Number} index Player index
```

Comment:
where does the term "player" come from? can we use "participant" in our naming to be consistent with docs/other implementations